### DISCUSSION TITLE
What is the difference between model and model no. field?

### DISCUSSION THREAD

**Discussion by RippieUK** [snipe/snipe-it#9888] [Q&A]
Hey people,

What is the difference between the 2 fields 'Model' and 'Model No.' ??

For example Dell Latitude 7380. Is the model and model no. not the same??

**Comment by PlaneNuts** [comment]
I think this would be a good example. Sometimes the same model name can have multiple model numbers depending on the country it was released in, or specific features

Model: iPhone 12
Model Number: A2172

or

Model: Galaxy S21
Model Number: SM-G991B
